postgres/codenamemap.yaml

```diff
@@ -76,10 +88,20 @@
 {{ debian_codename('focal', '12') }}
 
 ## Fedora
-# http://apt.postgresql.org/pub/repos/yum/reporpms/
+# https://download.postgresql.org/pub/repos/yum/13/fedora/
 # https://apps.fedoraproject.org/packages/postgresql
-# `oscodename` grain has long distro name
-{{ fedora_codename('Fedora-31', '11', 'Fedora 31 (Thirty One)') }}
-{{ fedora_codename('Fedora-30', '11', 'Fedora 30 (Thirty)') }}
-
-# vim: ft=sls
+{{ fedora_codename('Fedora-34', '13') }}
+{{ fedora_codename('Fedora-33', '13') }}
+
+## openSUSE
+# https://download.postgresql.org/pub/repos/zypp/13/suse/
+{{ opensuse_codename('openSUSE Leap 15.3', '13') }}
+{{ opensuse_codename('openSUSE Leap 15.2', '13') }}
+
+## Amazon
+Amazon Linux 2:
+  pkgs_deps:
+    - libicu
+    - systemd-sysv
+  pkg_repo:
+    baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/redhat/rhel-7-$basearch'
```

Comment:
Took my own advice given in https://github.com/saltstack-formulas/postgres-formula/pull/314#discussion_r664455489 and moved this `Amazon` block here.  Added the `pkgs_deps` override, as mentioned in https://github.com/saltstack-formulas/postgres-formula/pull/314#issuecomment-874786180.

Reply:
@myii thanks for the follow-up - yes libicu is required to install PG however we did it outside of the formula so it didn't point my attention.
+ for these changes. Let us wait for @vutny to perform the review and approve changes.